tools/clang/unittests/SPIRV/CodeGenSPIRVTest.cpp

```diff
@@ -1273,34 +1273,34 @@ TEST_F(FileTest, SpirvInterpolationError) {
 }
 
 TEST_F(FileTest, SpirvLegalizationOpaqueStruct) {
-  runFileTest("spirv.legal.opaque-struct.hlsl", Expect::Success,
-              /*runValidation=*/true, /*relaxLogicalPointer=*/true);
+  setRelaxLogicalPointer();
+  runFileTest("spirv.legal.opaque-struct.hlsl", Expect::Success);
```

Comment:
Expect::Success is the default value, so you can remove it. Also for the following cases.

Reply:
Good point!

Reply:
Done